econet_heatpump_water_heater.yaml

```diff
@@ -98,6 +102,20 @@ sensor:
     device_class: "energy"
     state_class: "total_increasing"
     entity_category: "diagnostic"
+  - platform: template
+    name: "Total Daily Energy"
```

Comment:
What's the benefit of this when the same can easily be done in Home Assistant? Now we need a global, a complicated logic here with a static variable and relying on syncing time with Home Assistant and at midnight resetting it.

Reply:
We've disabled time syncing because of reliability issues though, see https://github.com/esphome-econet/esphome-econet/pull/367